modules/audiencerunBidAdapter.js

```diff
@@ -44,19 +42,53 @@ function getBidFloor(bid) {
     });
     return bidFloor.floor;
   } catch (_) {
-    return 0
+    return 0;
   }
 }
 
+/**
+ * Returns the most top page referer.
+ *
+ * @returns {string}
+ */
+function getPageReferer() {
```

Comment:
Is there a reason this is being duplicated?

`bidderRequest.refererInfo.referer` does this logic already, so not really sure this is needed.

https://github.com/prebid/Prebid.js/blob/master/src/refererDetection.js

Reply:
Hi @robertrmartinez, it is not a duplicate as `bidderRequest.refererInfo.referer` returns only the URL of the page where Prebid.js is called while we need the URL that linked to this page and not the URL of the current page.

Here is a screen that shows it and is available at https://7buoei.csb.app/?pbjs_debug=true

![image](https://user-images.githubusercontent.com/57719351/162687362-8e0118e2-f4aa-4aed-afda-1bdf9dd756dd.png)

Feel free if you need more details.